### PR TITLE
Max send message len may exceed u32

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
Fix this [issue](https://github.com/pingcap/tikv/issues/3022).

### CODE REVIEW

**Comment by BusyJay** [comment]
`usize` is equal to `u64` when word size is 64bit.

**Comment by hicqu** [comment]
I also meet this problem. Seems `ChannelArguments` in grpc will truncate u64 as int32.

File: src/server/server.rs [hunk 1]
```diff
@@ -37,6 +38,7 @@ use super::{Config, Result};
 
 const DEFAULT_COPROCESSOR_BATCH: usize = 256;
 const MAX_GRPC_RECV_MSG_LEN: usize = 10 * 1024 * 1024;
+const MAX_SEND_MESSAGE_LEN: usize = 2 >> 30;
```
Comment (overvenus) [inline src/server/server.rs]: I think you mean `<<`.

**Comment by BusyJay** [comment]
This is a bug of grpcio, let's fix it there.

**Comment by BusyJay** [comment]
Not yet.

**Comment by siddontang** [comment]
Can we close it now? @BusyJay

**Pull request closed without merging**